### ISSUE TITLE
Start-PSUServer commented out in Universal.psm1 version 2.5.0

### ISSUE THREAD

**Issue by liquorichris** [ironmansoftware/powershell-universal#680]
### Steps to Reproduce

```
Get-Module universal -ListAvailable

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     2.5.0                 Universal                           Desk      {Add-UAScriptTag, New-PSUDashboard, Add-UDDashboardComponent, Add-UDDashboardFramework…}
```

```
Import-Module -Name Universal
Start-PSUServer -Port 5000
```

### Expected behavior

```console
PowerShell Universal Server should start on port 5000 and run in localhost.

`Universal.psm1` has `Start-PSUServer` commented out and not being Exported during the `Import-Module` process. Uncommenting the lines and re-import fixes the issue. 

Line 1: <# <--- Start of comment block
Line 73: #> <--- End of comment block
```

### Actual behavior

```console
Universal module returns the following error:

Start-PSUServer: The term 'start-psuserver' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

### Environment data

Version: 2.5.0

**Comment by adamdriscoll** [comment]
Looks like a problem in 2.4.1 as well. We'll be doing a 2.5.1 build since we have an issue with Windows Auth so I should be able to get this in there.

**Comment by liquorichris** [comment]
Awesome. Thanks!

**Comment by liquorichris** [comment]
@adamdriscoll Looks like there seems to be an issue with Swagger as well... Getting the following error:

`Fetch errorInternal Server Error /swagger/v1/swagger.json`

![image](https://user-images.githubusercontent.com/19211654/140971993-0dc6d780-afca-4c97-9831-f5c9c0e5b96d.png)

**Comment by adamdriscoll** [comment]
I've got both of these issues patched in the 2.5.1 branch. I think I'll do a release later today or tomorrow to take care of these.

**Comment by adamdriscoll** [comment]
Released in 2.5.1
